boxoffice/views/admin_report.py

```diff
@@ -27,6 +28,13 @@ def admin_report(item_collection):
     (ItemCollection, {'id': 'id'}, 'item_collection'),
     permission='org_admin')
 def tickets_report(item_collection):
-    headers = ['ticket id', 'invoice no', 'ticket type', 'base amount', 'discounted amount', 'final amount', 'discount policy', 'discount code', 'buyer fullname', 'buyer email', 'buyer phone', 'attendee fullname', 'attendee email', 'attendee phone', 'attendee details', 'utm_campaign', 'utm_source', 'utm_medium', 'utm_term', 'utm_content', 'utm_id', 'gclid', 'referrer']
+    headers = ['ticket id', 'invoice no', 'ticket type', 'base amount', 'discounted amount', 'final amount', 'discount policy', 'discount code', 'buyer fullname', 'buyer email', 'buyer phone', 'attendee fullname', 'attendee email', 'attendee phone', 'attendee details', 'utm_campaign', 'utm_source', 'utm_medium', 'utm_term', 'utm_content', 'utm_id', 'gclid', 'referrer', 'date']
     rows = LineItem.fetch_all_details(item_collection)
-    return csv_response(headers, rows)
+
+    def row_handler(row):
+        row_list = list(row)
+        # localize datetime
+        row_list[-1] = format_datetime(localize_timezone(row_list[-1]), format='long', locale=get_locale() or 'en')
```

Comment:
You shouldn't need `get_locale() or 'en'`. The function should supply a default.